app/graphql/enums/digital_object/order_fields_enum.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Enums::DigitalObject::OrderFieldsEnum < Types::BaseEnum
+  value 'RELEVANCE', 'sorting by relevance score', value: 'score'
```

Comment:
The all-caps sort field names stood out to me as an uncommon practice, based on other APIs I've used.  Is it okay if we go for lower case?  Or is there a reason that you prefer caps?  Also, you have lower-case asc and desc values in params.rb (`VALID_SORT_DIRECTION = ['asc', 'desc'].freeze`).

Reply:
One other thought about the translation of these enums to fields like `updated_at_dtsi`: It seems like we're coupling GraphQL too tightly with solr here.  I'm wondering if the translation to solr fields should happen elsewhere.